openedx/features/learner_profile/views/learner_profile.py

```diff
@@ -42,10 +45,34 @@ def learner_profile(request, username):
         GET /account/profile
     """
     try:
-        return render_to_response(
+        context = learner_profile_context(request, username, request.user.is_staff)
+        # TODO: LEARNER-2554: 09/2017: Remove message and cookie logic when we no longer want this message
+        dismiss_message = False
+        if (context['own_profile'] and
+                BOOST_PROFILE_VISIBILITY.is_enabled() and
+                request.COOKIES.get('dismiss-profile-message', '') != 'True'):
```

Comment:
Use ``not foo`` rather than ``foo != True``.

Reply:
I'd name the cookie for the current state, rather the action that was done. Maybe 'profile-message-viewed' since this is set whether or not the user dismisses it.

Reply:
I will change the cookie name, but I don't think I can use `not foo` as the check, because the cookie value will be a string, in this case `'True'`.

Reply:
Good point. I thought there was a helper function for handling boolean parameters like this, but I can't find one.